nimbus.go

```diff
@@ -3,15 +3,33 @@
 package main
 
 import (
-	statusnim "github.com/status-im/status-nim"
-	"runtime"
+	"crypto/ecdsa"
+
+	"github.com/jroimartin/gocui"
+	nimbusbridge "github.com/status-im/status-protocol-go/bridge/nimbus"
+	whispertypes "github.com/status-im/status-protocol-go/transport/whisper/types"
 )
 
 func init() {
-	runtime.LockOSThread()
+	nimbusbridge.Init()
+}
+
+func startNimbus(privateKey *ecdsa.PrivateKey, listenAddr string, staging bool) error {
```

Comment:
I'm curious about the security implications of this, i.e. using private key to start a separate process. Any thoughts?

If there are any or it isn't clear, we can leave a note/issue and get back to it later. Just jumped out at me.

Reply:
I don't think it's a separate process.

Reply:
You are right, shared lib. Still, memory usage and so on? To me this is too much of a black box from a security POV that I can't confidently say this isn't a problem. Are you certain it is equivalent to the existing code in Geth related to private key lifecycles and memory access?

Reply:
If we are concerned about it, maybe we should use a statically linked library instead of the dynamically linked one?

Reply:
There are issues with the statically linked library and different version of secp in geth and nimbus...was a bit of a rabbit hole. Perhaps worth revisiting later if there are other things that would be improved by it, but for now I wouldn't bother. For now, what about a comment in the code regarding it?

Reply:
Yeah, ran into those issues. Will add a comment then.